### PR TITLE
Shazam catalog

### LINKED ISSUE #115: Create the custom catalog for the shazam API

### PR DESCRIPTION
**Pull request by yasmineben510**
created a custom catalog to use shazam locally, it contains three songs against which audio can be matched. It was tested manually and worked correctly, I added some unit tests too but there not really useful compared to manual testing.

I also added the shazam button to the navigationBar, it informs the user about the recognition workflow through toasts since the idea is that the shazaming will work in the background while taking a picture.

I also refactored the code a little and modified the color of the adding post button.

This PR closes #115

### CODE REVIEW

File: app/src/androidTest/java/com/github/orkest/navigationTabTest.kt [hunk 1]
```diff
@@ -18,7 +19,7 @@ class navigationTabTest {
     @Test
     fun canAccessEveryNavigationTabFromMainActivity(){
         composeTestRule.setContent {
-            NavigationBar.CreateNavigationBar(navController = rememberNavController(),"")
+            NavigationBar.CreateNavigationBar(navController = rememberNavController(),"", MainActivity())
```
Comment (emilehreich) [inline app/src/androidTest/java/com/github/orkest/navigationTabTest.kt:22]: This additional button is temporary while waiting for the picture functionality right?

File: app/src/androidTest/java/com/github/orkest/shazam/data/AudioRecognitionTest.kt
Comment (emilehreich) [inline app/src/androidTest/java/com/github/orkest/shazam/data/AudioRecognitionTest.kt:1]: Great Tests!

File: app/src/main/java/com/github/orkest/shazam/data/AudioRecognition.kt
Comment (emilehreich) [inline app/src/main/java/com/github/orkest/shazam/data/AudioRecognition.kt:1]: Everything looks good to me, well documented! The hardcoded catalog is a nice alternative to the paid subscription to apple developers. Thank you.

File: app/src/main/java/com/github/orkest/shazam/data/AudioRecording.kt [hunk 2]
```diff
@@ -78,6 +79,7 @@ class AudioRecording {
                     val byteArray = readBuffer.sliceArray(0 until actualRead)
                     val audioChunk = AudioChunk(byteArray, actualRead, System.currentTimeMillis())
                     emit(audioChunk)
+                    delay(10)
```
Comment (emilehreich) [inline app/src/main/java/com/github/orkest/shazam/data/AudioRecording.kt:82]: what's the delay for?

**Review by emilehreich — APPROVED** [review]

File: app/src/androidTest/java/com/github/orkest/navigationTabTest.kt (hunk 1 above)
Comment (yasmineben510) [reply app/src/androidTest/java/com/github/orkest/navigationTabTest.kt:22]: Yes exactly, the goal would be that when we would get a button to access the camera which at the same time launches the shazaming (which is also why I chose to use toasts messages to display if a match happened or not ..)

File: app/src/androidTest/java/com/github/orkest/shazam/data/AudioRecognitionTest.kt
Comment (yasmineben510) [reply app/src/androidTest/java/com/github/orkest/shazam/data/AudioRecognitionTest.kt:1]: Thank you!

File: app/src/main/java/com/github/orkest/shazam/data/AudioRecording.kt (hunk 2 above)
Comment (yasmineben510) [reply app/src/main/java/com/github/orkest/shazam/data/AudioRecording.kt:82]: Without it there is too much work on the main thread and the app crashes, the delay allows for enough time to transmit the chunk before recording and storing another

**Pull request merged**